pkg/challenge/challenge.go

```diff
@@ -174,3 +193,34 @@ func Generate(signer crypto.Signer, challengeId []byte, challengeData []byte, re
 
 	return outerChallengeBytes, privEncKey, nil
 }
+
+func publicEcdsaToDer(key *ecdsa.PublicKey) ([]byte, error) {
```

Comment:
I think we should expose this in echelpers

Reply:
done